frontend/public/components/alerting.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import {HorizontalNav} from './utils';
+import {MonitoringUI, HeaderAlertmanagerLink} from './monitoring';
+
+const pages = [{
+  href: 'alerts',
+  name: 'Alerts',
+  primary: true,
+  component: MonitoringUI,
+}, {
+  href: 'silences',
+  name: 'Silences',
+  component: MonitoringUI,
+}, {
+  href: 'alertmanageryaml',
+  name: 'YAML',
+  component: MonitoringUI,
```

Comment:
I think you can use `component: AlertManagerYAML` here, which should prevent the Monitoring UI poller being run unnecessarily while on the YAML page.

Reply:
fixed